lib/index.js

```diff
@@ -1000,6 +1002,8 @@ _.use = use(Stream, _);
 addToplevelMethod('of', function (x) {
     return _([x]);
 });
+Stream.prototype[fl.of] = Stream.prototype.of;
```

Comment:
`fl.of` should be placed on the `Stream` directly, and not on its prototype.

Reply:
This will also be the case for `empty` in #657, I assume.

per https://github.com/fantasyland/fantasy-land/#empty-method:
> _Given a value m, one can access its type representative via the `constructor` property:_
> _`m.constructor.empty()`_

Reply:
Yes.